DESCRIPTION

```diff
@@ -15,7 +15,6 @@ Depends:
     scales,
     dplyr
 Imports:
-    vegan,
```

Comment:
In `mobsim` I neve had problems with the `vegan` dependency. But I think in general the less dependencies, the better

Reply:
yea I love vegan but it makes our package more likely to break in the future as 3rd party code changes.